openapi.yml

```diff
@@ -441,7 +443,6 @@ components:
               description: Status of the descriptive element value relative to other instances
                 of the element.
               type: string
-              # type: integer (uncomment when issue #154 is resolved)
```

Comment:
does this one need a an "order" property added?

Reply:
No, there hasn't been a use case for that yet, so I decided to leave it out for now.